x/protorev/keeper/epoch_hook.go

```diff
@@ -54,6 +60,47 @@ func (h EpochHooks) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epoch
 	return nil
 }
 
+// CalculateDistributeProfits is executed after epoch. It gets the current base denom profits and distributes them.
+func (h EpochHooks) CalculateDistributeProfits(ctx sdk.Context) error {
```

Comment:
Nit: the current name makes me think this function has no side-effects. Renaming makes it clearer that distributing happens here

```suggestion
func (h EpochHooks) CalculateAndDistributeProfits(ctx sdk.Context) error {
```

Reply:
Good idea updated here => https://github.com/osmosis-labs/osmosis/pull/8106/commits/e043b86ebc451eb183f8de6fc70e26955c1fb29d